lib/rdf/transaction.rb

```diff
@@ -122,43 +170,21 @@ def writable?
     end
 
     ##
-    # Returns `false` to indicate that this transaction is append-only.
-    #
-    # Transactions do not support the `RDF::Enumerable` protocol directly.
-    # To enumerate the RDF statements to be inserted or deleted, use the
-    # {RDF::Transaction#inserts} and {RDF::Transaction#deletes} accessors.
+    # Returns `true` if this is a read/write transaction, `false` otherwise.
     #
     # @return [Boolean]
-    # @see    RDF::Readable#readable?
-    def readable?
-      false # TODO: support RDF::Enumerable and RDF::Queryable.
+    # @see     RDF::Writable#mutable?
+    def mutable?
```

Comment:
Depending on the outcome of the discussion at https://github.com/ruby-rdf/rdf/pull/250/files#r49671230, we may want to consider removing `writable?` and `mutable?`